webservice/template/admin/logs/index.tmpl

```diff
@@ -0,0 +1,191 @@
+{{define "title"}}Logs{{end}}
+{{define "head"}}
+    <link rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.14.2/dist/bootstrap-table.min.css">
+    <style>
+        .id-col {
+            font-weight: 700;
+        }
+    </style>
+{{end}}
+{{define "content"}}
+    <h2 class="display-4 mb-3">Logs</h2>
+    <hr>
+
+    <div class="row">
+        <div class="col">
+            {{ if eq ( len .Logs ) 0}}
+                <h4 class="display-6 mb-3">No logs available</h4>
+            {{ else }}
+
+                <table class="table table-hover table-striped table-sm" id="table" data-toggle="table"
+                       data-pagination="true" data-search="true" data-page-size="100">
+                    <thead class="thead-light">
+                    <tr>
+                        <th data-sortable="true" scope="col">#</th>
```

Comment:
What is #? (From client perspective)
Number of log, user, activity?
Rename for convenience

Reply:
changed to  `User ID`